convert2rhel/backup/packages.py

```diff
@@ -143,11 +144,15 @@ def enable(self):
                 loggerinst.info("The repository directory %s seems to be empty or non-existent.")
                 self.reposdir = None
 
+        disable_repos = ["rhel*"] + tool_opts.enablerepo
```

Comment:
I think we should move this to the __init__ to allow the caller to override the disable_repos if needed. This way, we will always assume that rhel*+ enablerepo will be disabled for all package backup, regardless of what we choose. 

Maybe we could improve it like this: 

```python
def __init__(self, pkgs, reposdir=None, set_releasever=False, custom_releasever=None, varsdir=None, disable_repos=None):
        """
        Keep control of system packages before their removal to backup and
        restore in case of rollback.
        :param pkgs list[str]: List of packages to backup.
        :param reposdir str: If a custom repository directory location needs to
            be used, this parameter can be set with the location for it.
        :param set_releasever bool: If there is need to set the relesever while
            downloading the package with yumdownloader.
        :param custom_releasever str: Custom releasever in case it need to be
            overwritten and it differs from the `py:system_info.releasever`.
        :param varsdir str: Location to the variables directory in case the
            repository files needs to interpolate variables from those folders.
        :param disable_repos list[str]: Repositories to disable during package download. Defaults to None.
        """
        super(RestorablePackage, self).__init__()
        self.pkgs = pkgs
        self.reposdir = reposdir
        self.set_releasever = set_releasever
        self.custom_releasever = custom_releasever
        self.varsdir = varsdir
        self.disable_repos = disable_repos or ["rhel*"] + tool_opts.enablerepo
        self._backedup_pkgs_paths = []
```

Then later while we are downloading the packages, it would use it like this: 

```python
        for pkg in self.pkgs:
            self._backedup_pkgs_paths.append(
                utils.download_pkg(
                    pkg=pkg,
                    dest=BACKUP_DIR,
                    # RHELC-884 disable the RHEL repos to avoid downloading pkg from them
                    disable_repos=self.disable_repos,
                    set_releasever=self.set_releasever,
                    custom_releasever=self.custom_releasever,
                    varsdir=self.varsdir,
                    reposdir=self.reposdir,
                )
            )
```